deployment/live/gcp/conformance/README.md

```diff
@@ -24,27 +17,87 @@ need to be manually applied.
 
 ## Manual deployment 
 
-First authenticate via `gcloud` as a principle with sufficient ACLs for
-the project:
+### Prerequisites
+
+You'll need the following tools installed:
+
+- [`docker`](https://docs.docker.com/engine/install/)
+- [`gcloud`](https://cloud.google.com/sdk/docs/install)
+- One of:
+   + [`terraform`](https://developer.hashicorp.com/terraform/install) or
+   + [`opentofu`](https://opentofu.org/docs/intro/install/)
+- [`terragrunt`](https://terragrunt.gruntwork.io/docs/getting-started/install/)
+
+
+### Process
+
+First ensure you've got the right Google Cloud project configured, and authenticate via `gcloud`
+as a principle with sufficient ACLs for the project:
 ```bash
+gcloud config get project
+gcloud config set project {YOUR PROJECT}
```

Comment:
There is a step to export the `GOOGLE_PROJECT` enviornment variable later. Maybe we can move that above this line, so this command can reuse the `${GOOGLE_PROJECT}`

```
export GOOGLE_PROJECT={VALUE}
```

Reply:
I've restructured the doc a bit, is it better now?